src/components/Comments/Comment.js

```diff
@@ -179,6 +252,13 @@ const Comment = ({
             </ButtonHightlight>
           </TouchableOpacity>
         )}
+        <ButtonHightlight
+          onLongPress={handleOnLongPress}
+          style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
+          onPress={() => refSheet.current.open()}>
+          <MemoSendDM width={20} height={18} />
+        </ButtonHightlight>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new button is added here for opening the bottom sheet menu. The usage of `refSheet.current.open()` is correct and will open the bottom sheet when the button is pressed.
```diff
+ <ButtonHightlight
+   onLongPress={handleOnLongPress}
+   style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
+   onPress={() => refSheet.current.open()}>
+   <MemoSendDM width={20} height={18} />
+ </ButtonHightlight>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->